salt/utils/parsers.py

```diff
@@ -654,7 +661,9 @@ def __setup_logfile_logger_config(self, *args):  # pylint: disable=unused-argume
             self.config.pop(self._logfile_loglevel_config_setting_name_)
```

Comment:
Also, the `cli_log_path` below appears to be completely unused, I remember I have removed it in previous revision.

Right after that there is another popping from config for `log_file` key. It should not affect anything, but that's totally unnecessary.

Everything else looks good.